crates/cli-support/src/js/mod.rs

```diff
@@ -469,7 +469,8 @@ impl<'a> Context<'a> {
             OutputMode::Web => {
                 self.imports_post.push_str("let wasm;\n");
                 init = self.gen_init(needs_manual_start, Some(&mut imports))?;
-                footer.push_str("export default init;\n");
+                footer.push_str("export { initSync }\n");
+                footer.push_str("export default init;");
```

Comment:
ℹ️ I have noticed there was two newlines at the end of the generated JS so I removed one newline.